client/platform/desktop/frontend/store/index.ts

```diff
@@ -19,7 +19,9 @@ export async function migrate() {
 export default function wrap(): Api {
   async function loadMetadata(datasetId: string) {
     const meta = await api.loadMetadata(datasetId);
-    setRecents(meta, (new Date()).toString());
+    if (!datasetId.includes('/')) { // Only update if not multiCam
+      setRecents(meta, (new Date()).toString());
+    }
```

Comment:
I can't because the meta for an individual Camera has no idea that it's a camera in a multiCam setup.  I may have to do more thinking about if each camera should know it's an individual camera and how to store that in `meta`.

This could be changed if was modified to `loadMetadata(datasetId: string, cameraId?: string)`